packages/form/addon/components/cf-field/input/action-button.js

```diff
@@ -12,8 +12,9 @@ if (macroCondition(dependencySatisfies("@projectcaluma/ember-workflow", ""))) {
       super(...args);
 
       assert(
-        "The document must have a `workItemUuid` for `<CfField::Input::ActionButton />` to work.",
-        this.args.field.document.workItemUuid,
+        "The document or context must have a `workItem` for `<CfField::Input::ActionButton />` to work.",
+        this.args.field.document.workItemUuid ||
+          this.args.context?.actionButtonWorkItemId,
```

Comment:
Assertion message must be more explicit to be helpful

Reply:
I tried to improve it a bit.